lib/diagnostic.js

```diff
@@ -32,7 +33,9 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({ name: "House Stark" }, {$push: {members: ['Ned Stark', 'Arya Stark', 'Sansa Stark']}});
```

Comment:
You can use a combination of `$set` and `$push` (with one argument or a nested object with an `$each` key) to add new fields here.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```